src/components/responsive-flyout/OverlayHeader.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as React from 'react';
+
+import classNames from 'classnames';
+import CloseButton from './CloseButton';
+import './styles/OverlayHeader.scss';
+
+export type Props = {
+    /** Components to render in the header */
+    children?: React.Node,
+    /** Set className to the overlay header */
+    className?: string,
+    /** Set click handler when close button is clicked */
+    onCloseClick?: Function,
+};
+
+const OverlayHeader = (props: Props) => {
+    const { children, onCloseClick, className } = props;
+
+    const handleClick = event => {
+        event.preventDefault();
+        event.stopPropagation();
+    };
+
+    return (
+        <div role="presentation" onClick={handleClick} className={classNames('overlay-header', className)}>
+            <div className="oh-content">{children}</div>
+            <CloseButton className="oh-close-btn" onClick={onCloseClick} />
```

Comment:
Is this class implemented?

Reply:
Yes, CloseButton.js. I'm planning on moving this out to it's own module to be reused across other components